com.redhat.tuned.policy

```diff
@@ -247,4 +247,24 @@
     </defaults>
   </action>
 
+  <action id="com.redhat.tuned.instance_create">
```

Comment:
I think this requires auth_admin otherwise regular user could reconfigure the machine. I.e. the rule of thumb is: any user can list config, but only authorized user (admin) can make changes because it could cause DOS and have impact on the system performance.

Reply:
I think both instance_create/instance_destroy should require auth_admin.

Reply:
Even now when the instance is empty and attach/detach devices requires auth_admin, thus instance_create/destroy shouldn't cause any harm, it's change of the configuration and as such it should require auth_admin.

Reply:
The calls now require auth_admin (and the feature now has its own separate PR #596)